wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -203,8 +203,8 @@
           },
           "id": {
             "type": "string",
-            "description": "Describes the component FQN",
-            "pattern": "^[a-zA-Z0-9_\\-\\./]{1,}:[a-zA-Z0-9_\\-\\.]{1,}$",
+            "description": "Describes the component id. Component id it's fqn package from plugin registry or modified gist url link to the yaml file with component definition.",
```

Comment:
IMHO the mention of the gist is a little bit too specific. This could be a URL to a completely random location of a random server.

Reply:
`gist` is replaced with custom plugin registry